### DISCUSSION TITLE
cant download playlist

### DISCUSSION THREAD

**Discussion by ghost** [KRTirtho/spotube#611] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

the entire 3 dot menu of a playlist is grayed out so i cant download anything only one by one
ive tried the normal version the nightly and on f-droid

### Expected Behavior

not to be grayed out so i can download entire playlist

### Steps to reproduce

1. open the app
2. goto liberary
3. goto playlist
4. click 3 dots
5. the menu is grayed out

### Operating System

Ubuntu 22.04.2 LTS

### Spotube version

3.0.0

### Installation source

GitHub Actions (Nightly Binary), F-Droid (Android), Flathub (Flatpak)

**Comment by KRTirtho** [comment]
You've to first select any/all tracks from the playlist by long-pressing any track

**Reply by KRTirtho** [reply]
Only, after that those action will become enabled
